.env.example

```
@@ -0,0 +1,30 @@
+VPN_INTERFACE=wg0
+
+# Twilio Verify API (for OTP authentication via SMS, WhatsApp, Email)
+# Create a Verify Service in Twilio Console: https://console.twilio.com/us1/develop/verify/services
+TWILIO_ACCOUNT_SID=your-account-sid
+TWILIO_AUTH_TOKEN=your-auth-token
+TWILIO_VERIFY_SERVICE_SID=your-verify-service-sid
+TWILIO_PHONE_NUMBER=+1234567890
+
+# Age x25519 recipients (public keys; safe to embed).
+# The operator holds the matching private keys offline.
+AGE_RECIPIENT_QUESTIONNAIRE=age1...   # questionnaire answers + gate-submission email
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)